### ISSUE TITLE
Error: Breadcrumb name "this.that.path" has already been registered, Laravel 11.33

### ISSUE THREAD

**Issue by rohankleem** [diglactic/laravel-breadcrumbs#84]
PHP 8.2.12 — Laravel 11.33.2
diglactic/laravel-breadcrumbs v9.0.0
Can't get breadcrumbs working at all.

**Internal Server Error**
Diglactic\Breadcrumbs\Exceptions\DuplicateBreadcrumbException
**Breadcrumb name "this.that.path" has already been registered**
C:\xampp\htdocs\website-pay\Pay\vendor\diglactic\laravel-breadcrumbs\src\Manager.php :62

I don't appear to be loading routes/breadcumbs.php twice as this would imply, this isn't the cause (no other inclusion of routes/breadcumbs.php in app)
Issues occurs when 
`Breadcrumbs::for .... `
is added to breadcumbs.php

routes/breadcumbs.php
```
<?php
use Diglactic\Breadcrumbs\Breadcrumbs;

dump ("When I add the lines below, I am run twice. When removed I am only run once");
Breadcrumbs::for('this.that.path', function ($trail) {
    $trail->push('Payment Requests', route('this.that.path'));
});
```

AppServiceProvider.php
```
<?php
namespace App\Providers;
use Illuminate\Support\Facades\App; 
use Illuminate\Support\ServiceProvider;
class AppServiceProvider extends ServiceProvider
{
    public function register(): void
    {
        //
    }
    public function boot(): void
    {
       dump ("I am only running once");
        require_once base_path('routes/breadcrumbs.php');
    }
}
```

**Comment by shengslogar** [comment]
Why are you manually requiring `routes/breadcrumbs.php`? This isn't part of the [setup instructions](https://github.com/diglactic/laravel-breadcrumbs?tab=readme-ov-file#getting-started).

**Comment by rohankleem** [comment]
Thanks, I did the setup using the instructions, but wasn't getting any breadcrumbs being displayed when I was testing. I did some testing and couldn't find any evidence of routes/breadcrumbs.php being included, so then did some research and found somewhere saying to include, and this made sense to me that it would be required. But apparently not. I will reset up and retest again, going through the setup instructions again. Thanks for getting back to me.
